fe/fe-core/src/main/java/com/starrocks/alter/LakeTableAlterMetaJob.java

```diff
@@ -205,6 +205,11 @@ protected void runFinishedRewritingJob() throws AlterCancelException {
                     tempProperties.put(PropertyAnalyzer.PROPERTIES_ENABLE_PERSISTENT_INDEX, String.valueOf(metaValue));
                     GlobalStateMgr.getCurrentState().getLocalMetastore()
                             .modifyTableMeta(db, table, tempProperties, metaType);
+                } else if (metaType == TTabletMetaType.DATACACHE_ENABLE) {
```

Comment:
Only need to update the shard info in StarMgr, no need to create this `LakeTableAlterMetaJob`